src/HasNotes.php

```diff
@@ -17,6 +17,14 @@ public function setNote(string $note): Note
         return $this->notes()->create(['note' => $note]);
     }
 
+    public function setNoteWithGroup(array $note): Note
```

Comment:
Seeing this now, I'm not sure this is more helpful than just calling `$this->notes()->create(...)`. 

What do you think?

Reply:
@james2doyle Yeah I agree, I can remove it and just use the existing notes API there